hoot-core/src/main/cpp/hoot/core/algorithms/alpha-shape/AlphaShape.cpp

```diff
@@ -187,16 +182,16 @@ std::shared_ptr<Polygon> AlphaShape::_convertFaceToPolygon(const Face& face) con
   return result;
 }
 
-bool AlphaShape::_isBoundary(const Edge& e) const
+bool AlphaShape::_isBoundary(const double alpha, const Edge& e) const
```

Comment:
Passing the alpha value around instead of using `_alpha` will allow us to try different values in the "auto alpha" algorithm without changing the member variable's value.